python/GafferArnoldUI/ArnoldLightFilterUI.py

```diff
@@ -101,6 +117,34 @@ def __activateRoundness( parent ) :
 
 		],
 
-	}
+		"parameters.width_edge" : [
+
+			"layout:visibilityActivator", "falloffActivator",
```

Comment:
Using "visibilityActivator" is making a confusingly empty Falloff section. I suggest we just use a regular "activator" for now, although at some point we should perhaps hide empty sections.

Reply:
Done here: de31e60